scripts/sct_propseg.py

```diff
@@ -15,6 +15,66 @@
 import sys
 import sct_utils as sct
 import os
+from scipy import ndimage as ndi
+import numpy as np
+from sct_image import orientation
+
+
+def check_and_correct(fname_segmentation, fname_centerline, threshold_distance=5.0, verbose=0):
+    # convert segmentation image to RPI
```

Comment:
Use the docstring format to document the function.
`"""Convert segmentation image to RPI"""`

I use this as reference: [https://google.github.io/styleguide/pyguide.html#Comments](Google style guide)